LiskHQ/lisk-mobile
# Update to React Native 0.59 - Closes #835

# What was the bug or feature?
Described in #835.

### How did I fix it?
- [x] Updated iOS.
- [x] Updated Android.

## Type of change
- Enhancement (a non-breaking change which adds functionality)

## How to test it
Consider the following points:
- App should present no issues on iOS or Android.
- iOS build should also be able to run from CLI.
- Android build should be able to run without manually changing the `settings.gradle`.
- Deprecated/extracted components should have been installed and imported from the new repo, e.g. `AsyncStorage`.
- README should be consistent with the last changes.
- Make sure that it is possible to push commits to remote. Sometimes some of the jest tests have failed for no reason and `node_modules` need to be removed in order to push. Could be related to [this](https://stackoverflow.com/questions/45117415/travis-ci-jest-preset-react-native-not-found) and [this](https://github.com/facebook/metro/issues/242) but could also be a local issue.

# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
